pageserver/src/tenant/timeline.rs

```diff
@@ -1076,23 +1075,7 @@ impl Timeline {
 
         if let Some((metric, start)) = start {
             let elapsed = start.elapsed();
-            let ex_throttled = if let Some(throttled) = throttled {
-                elapsed.checked_sub(throttled)
-            } else {
-                Some(elapsed)
-            };
-
-            if let Some(ex_throttled) = ex_throttled {
-                metric.observe(ex_throttled.as_secs_f64());
-            } else {
-                use utils::rate_limit::RateLimit;
-                static LOGGED: Lazy<Mutex<RateLimit>> =
-                    Lazy::new(|| Mutex::new(RateLimit::new(Duration::from_secs(10))));
-                let mut rate_limit = LOGGED.lock().unwrap();
-                rate_limit.call(|| {
-                    warn!("error deducting time spent throttled; this message is logged at a global rate limit");
-                });
-            }
+            metric.observe(elapsed.as_secs_f64());
```

Comment:
The metric descriptions should be updated now that we are not excluding the throttle time.

Reply:
Why are we doing this though? I understand why we're including the throttle in the smgr metrics, but why not keep the timeline metrics as they were? Feels like i'm missing something.

Reply:
Fair point.

Reply:
be2d64bee396b262ae187bf04c427fd24065eaf4